src/helpers/getLocaleFromRequest/spec.js

```diff
@@ -47,6 +47,13 @@ describe('The `getLocaleFromRequest` helper', function () {
     expect(actual).toEqual(expected)
   })
 
+  it('should not throw if failing to decode query from request', function () {
+    var request = getMockedRequest({ query: { locale: '/at//%c0%ae%c0%ae%c0%ae%c0%c0%a%c0%c0%c0%c0%af%c0%af%c0%af%c0%c0%ae%c0%c0%ae%c0%ae%c0%afwindowswin.ini' } })
```

Comment:
😄

Reply:
This is an example of URL used to DDoS us.

Reply:
I know. :)